cocoviewer.py

```diff
@@ -798,4 +812,4 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    main()
```

Comment:
```suggestion
    main()

```
nit: newline at the end of a file